generative_ai/embeddings/multimodal_video_example.py

```diff
@@ -19,9 +19,9 @@
 
 
 def get_video_embeddings() -> MultiModalEmbeddingResponse:
```

Comment:
The [documentation](https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-multimodal-embeddings#vid-embedding) for this sample says:

> Use the following sample to get embeddings for video content **alone**.

I believe the goal for this sample should be to show that the user can still use the `MultiModalEmbeddingModel` class to get one type of embeddings (e.g., video-only)